packages/at_client/lib/src/encryption_service/abstract_atkey_encryption.dart

```diff
@@ -201,59 +269,62 @@ abstract class AbstractAtKeyEncryption implements AtKeyEncryption {
   }
 
```

Comment:
renamed _storeSharedKey and removed `shouldSync` parameter as all of this stuff is no longer dependent on sync in any way